### PR TITLE
Relationship converter new

### PR DESCRIPTION
**Pull request by bmiller-jama**
Updated PR for Python Relationship Converter without all the Python dependencies/packages

### CODE REVIEW

File: get-pip.py
Comment (nmchale-jama) [inline get-pip.py:1]: delete this file from repo and rm from git

**Review by nmchale-jama — CHANGES_REQUESTED** [review]
Please remove the get pip .py f ile

**Comment by nmchale-jama** [comment]
Plase create a requiremetns.txt file via pip freeze

**Comment by nmchale-jama** [comment]
Also please upadte the readme to reflect new dependencies and how to install via pip

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+beautifulsoup4==4.9.3
+bs4==0.0.1
+certifi==2021.5.30
+chardet==4.0.0
+colorama==0.4.4
+halo==0.0.31
+idna==2.10
+log-symbols==0.0.14
+progress==1.5
+py-jama-rest-client==1.16.0
```
Comment (trobinson-jama) [inline requirements.txt]: Changed py rest client to 1.17.1

**Review by trobinson-jama — APPROVED** [review]

**Review by trobinson-jama — APPROVED** [review]

**Pull request merged**